### DISCUSSION TITLE
[CHECK] - Standardize devcontainer to what it installs/runs

### DISCUSSION THREAD

**Discussion by kjaymiller** [kjaymiller/cookiecutter-relecloud#172] [General]
Do we have a consistent idea of what the devcontainer has done and what needs to still be done by the developer.

This is important as we right docs for how to deploy. Some examples:

- When are the apps (`flask_app`, `fastapi_app`) installed
- When is the data seeded/migrated
- When are the testing tools installed (`requirements-dev.*`)
- When is playwright installed? What about `playwright install --with-deps`

**Comment by pamelafox** [comment]
The playwright install takes so long that I currently dont do it in devcontainer / postCreate. But it does mean you have to read readme before you can run tests. What do you think? We also logged it here: https://github.com/Azure-Samples/azure-django-postgres-aca/issues/27
